File: Practica 10/Java/Circulo.java
package Java;

public class Circulo extends Figura {
    private double radius;

    public Circulo(String color, double radius) {
        super(color);
        this.radius = radius;
    }

    @Override
    public double perimetro() {
        return 2 * Math.PI * radius;
    }

    @Override
    public double area() {
        return Math.PI * radius * radius;
    }
}

File: Practica 10/Python/Circulo.py
from Figura import Figura
import math

class Circulo(Figura):
    def __init__(self, color: str, radius: float):
        super().__init__(color)
        self.radius = radius
    
    def perimetro(self) -> float:
        return 2 * math.pi * self.radius
    
    def area(self) -> float:
        return math.pi * self.radius**2
